metriq_gym/process.py

```diff
@@ -58,20 +64,17 @@ def poll_job_results(jobs_file: str, job_type: BenchJobType) -> list[BenchJobRes
             job = get_job(result)
             status = job.status()
             
-            if (status == JobStatus.RUNNING) or (result.job_type != job_type):
-                # Still running
+            if (status not in JOB_FINAL_STATES) or (result.job_type != job_type):
```

Comment:
[nit]: parens are redundant (unneeded)

Reply:
I like the parens for better readability

Reply:
The parens shall live another day.